src/media-groups.js

```diff
@@ -149,6 +156,18 @@ export const onTrackChanged = (type, settings) => () => {
     return;
   }
 
+  if (activeGroup.masterPlaylist) {
+    if (activeTrack && lastTrack && activeTrack.id !== lastTrack.id) {
+      const mpc = settings.vhs.masterPlaylistController_;
+
+      mediaType.logger_(`track change. Switching master audio from ${lastTrack.id} to ${activeTrack.id}`);
+      masterPlaylistLoader.pause();
+      mainSegmentLoader.resetEverything();
+      mpc.fastQualityChange_();
```

Comment:
`fastQualityChange` should reset everything unless the media is the same. And if the media is the same, would we ever resume playback (the loader would pause but `fastQualityChange` would return early)?

Reply:
We will only get in here if the track id changed. which will cause us to select from a new set of playlists in fastQualityChange which will mean that the playlists should never be the same.

Reply:
updated the code to check that the playlist will actually change.